protocol/v1/qbft/instance/commit.go

```diff
@@ -13,14 +13,12 @@ import (
 	"github.com/bloxapp/ssv/protocol/v1/qbft"
 	"github.com/bloxapp/ssv/protocol/v1/qbft/instance/msgcont"
 	"github.com/bloxapp/ssv/protocol/v1/qbft/pipelines"
-	"github.com/bloxapp/ssv/protocol/v1/qbft/validation/signedmsg"
 )
 
 // CommitMsgPipeline - the main commit msg pipeline
 func (i *Instance) CommitMsgPipeline() pipelines.SignedMessagePipeline {
 	validationPipeline := i.CommitMsgValidationPipeline()
 	return pipelines.Combine(
-		signedmsg.ProposalExists(i.State()),
```

Comment:
Why remove this?

Reply:
@nivBlox spec uses different errors texts upon prepare and commit if proposal doesn't exist, so it's not possible to use a generic function. It's checked inside `ValidateProposal`. This issue appeared in this PR because I changed error mapping to reversed.